vcxagency-node/test/unit/notifications/event-adapter-redis.spec.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+/* eslint-env jest */
+global.LOG_LEVEL = process.env.LOG_LEVEL || 'info'
+global.LOG_JSON_TO_CONSOLE = process.env.LOG_JSON_TO_CONSOLE === 'true'
+global.SILENT_WINSTON = process.env.SILENT_WINSTON === 'false'
```

Comment:
This global variable seems to be set (in many tests) but never read.

Reply:
it's being used by `logger-builder.js`

```
createConsoleLogger(mainLoggerName, formatter, logLevel, process.env.SILENT_WINSTON === 'true')
```

as a mean for tests to have option opt out of application logs during test execution

Reply:
The global variable is not used on the line you quoted.

Reply:
Touche, updated logger file to read off `global` instead of `process.env` to stay consistent (it's reading other stuff such as `LOG_LEVEL` or `LOG_JSON_TO_CONSOLE` from `global`)